examples/sample_config.toml

```diff
@@ -0,0 +1,16 @@
+[server]
```

Comment:
Instead of making this a sample_config, what about making it a `template.toml` in the `config/` for now. I am generally not a fan of this kind of sample files :)

Reply:
updated